var/key_value_store/config.json

```diff
@@ -1 +1 @@
-{"tva":0.2,"logo":"s:87:\"https:\/\/n7consulting.fr\/themes\/IOTH-bootstrap-basic\/assets\/images\/logo_n7consulting.png\";","adresse":"s:46:\"2 rue Charles Camichel,  31071 Toulouse CEDEX7\";","url":"s:23:\"https:\/\/n7consulting.fr\";","domaineEmailEtu":"s:15:\"etu.enseeiht.fr\";","domaineEmailAncien":"s:18:\"almuni.enseeiht.fr\";","anneCreation":"s:4:\"1977\";","annee1Jeyser":"s:4:\"2015\";","presidentPrenom":"s:7:\"Antoine\";","presidentNom":"s:6:\"Dupond\";","presidentTexte":"s:15:\"Son Pr\u00e9sident,\";","tresorierPrenom":"s:6:\"Thomas\";","tresorierNom":"s:6:\"Dupont\";","tresorierTexte":"s:15:\"Son Tr\u00e9sorier,\";","nom":"s:13:\"N7 Consulting\";","abbr":"s:3:\"N7C\";","email":"s:23:\"contact@n7consulting.fr\";","gaTracking":null,"namingConvention":"s:3:\"nom\";"}
\ No newline at end of file
+{"tva":0.2,"logo":"s:87:\"https:\/\/n7consulting.fr\/themes\/IOTH-bootstrap-basic\/assets\/images\/logo_n7consulting.png\";","adresse":"s:46:\"2 rue Charles Camichel,  31071 Toulouse CEDEX7\";","url":"s:23:\"https:\/\/n7consulting.fr\";","domaineEmailEtu":"s:15:\"etu.enseeiht.fr\";","domaineEmailAncien":"s:18:\"almuni.enseeiht.fr\";","anneeCreation":"s:4:\"1977\";","annee1Jeyser":"s:4:\"2015\";","presidentPrenom":"s:7:\"Antoine\";","presidentNom":"s:6:\"Dupond\";","presidentTexte":"s:15:\"Son Pr\u00e9sident,\";","tresorierPrenom":"s:6:\"Thomas\";","tresorierNom":"s:6:\"Dupont\";","tresorierTexte":"s:15:\"Son Tr\u00e9sorier,\";","nom":"s:13:\"N7 Consulting\";","abbr":"s:3:\"N7C\";","email":"s:23:\"contact@n7consulting.fr\";","gaTracking":null,"namingConvention":"s:3:\"nom\";"}
```

Comment:
It's one of my commit.
For a not so good reason, but you might have a suggestion.

Config.json.dist is the model of file, copied by the install script. 
At least one of the value stored in config.json is injected into I don't remember which service. Thus when an external service (insights, travis..) build the project, the config.json should be there.

With travis, I can easily copy the script before launching tests. However, for sensiolabs insights, it has to be done in a composer pre-install script. Every time the apache docker container is launched, it executes `composer install` (and thus execute pre-install script, so would copy the dist file on the existing config file). That behaviour would mean that each time the web container is started, configuration is lost.

My workaround is to distribute both file, keep install script minimal (no tests to check if the file exists and thus don't depend on the OS, no need to create additional side effects) and don't track changes on config.json.